lib/net/pool.js

```diff
@@ -1883,6 +1894,142 @@ class Pool extends EventEmitter {
     }
   }
 
+  /**
+   * Handle peer `getcfilters` packet.
+   * @method
+   * @private
+   * @param {Peer} peer
+   * @param {GetCFiltersPacket} packet
+   */
+
+  async handleGetCFilters(peer, packet) {
+    if (this.options.selfish)
+      return;
+
+    if (this.chain.options.spv)
+      return;
+
+    if (this.chain.options.prune)
+      return;
+
+    if (this.chain.options.index)
+      return;
+
+    // The filter type should be valid
+    if (packet.filterType !== common.FILTERS.BASIC)
+      return;
+
+    const stopHeight = this.chain.getHeight(packet.stopHash);
+    // The stop hash doesn't exist
+    if (stopHeight == null)
+      return;
+
+    if (this.chain.getHeight(packet.stopHash) - packet.startHeight >= 1000)
+      return;
+
+    for (let i = packet.startHeight; i <= stopHeight; i++) {
+      const filter = await this.chain.getFilter(i);
```

Comment:
`chain.getFilter` doesn't exist yet right? Do we need to add something else to https://github.com/bcoin-org/bcoin/pull/1057 ? I expect this function to also accept a filter TYPE as parameter?

Reply:
Will add this once #1057 is merged